app/controllers/phones_controller.rb

```diff
@@ -3,7 +3,11 @@
 class PhonesController < ApplicationController
 
   def new
-    @phone = Phone.new
+    if current_account.level != 1
```

Comment:
not != but <. Cause person with level 2 can add another phone

Reply:
oh okay, sorry